Makefile

```
@@ -0,0 +1,105 @@
+##################################################
+# Variables                                      #
+##################################################
+SHELL           = /bin/bash
+
+IMAGE_REGISTRY 	?= ghcr.io
+IMAGE_REPO     	?= kedacore
+VERSION 		?= main
+
+IMAGE_OPERATOR 		?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-operator:$(VERSION)
+IMAGE_INTERCEPTOR	?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-interceptor:$(VERSION)
+IMAGE_SCALER		?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-scaler:$(VERSION)
```

Something we might consider is moving them to:
```suggestion
IMAGE_OPERATOR 		?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on/operator:$(VERSION)
IMAGE_INTERCEPTOR	?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on/interceptor:$(VERSION)
IMAGE_SCALER		?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on/scaler:$(VERSION)
```

---

Tracked in https://github.com/kedacore/http-add-on/issues/567

---

I can do it as part of the PR

---

let's discuss this in the issue